frontend/src/component/WeatherForecastCurrent.tsx

```diff
@@ -1,19 +1,53 @@
-import React from 'react'
+import React, {useState, useEffect} from 'react'
 
 import WeatherPanel from './WeatherPanel'
-
+import icons from './WeatherIcon.module.sass'
 import styles from './WeatherForecastCurrent.module.sass'
+import { NULL } from 'sass'
 
-export default function WeatherForecastCurrent() {
-    return (
-        <WeatherPanel className={styles.WeatherForecastCurrent}>
+const CLOUDY = "images/cloudy.png"
+const PARTLY_CLOUDY = "images/partly_cloudy.png"
+const RAINY = "images/rainy.png"
+const SNOWY = "images/snowy.png"
+const STORMY = "images/stormy.png"
+const SUNNY = "images/sunny.png"
+//, high: string, low: string, weather: string
+type MyProps = {city: string, unit: string, data:string[]};
+type MyState = { current_temp: number };
+export default class WFC extends React.Component<MyProps,MyState>{    
```

Comment:
We need to figure out how to convert forecast into one of these types. Also, this should probably be an enum, with a pairing to the image locations.

Reply:
Resolved by a672642.